### PR TITLE
chore - vue-cli init.

### PR DESCRIPTION
**Pull request by neverendingqs**
Converts the vue app to a "full" vue app with less hacky JS code wiring.

It's a bit heavier than a simple html/js/css, but less hacky. Thoughts?

### CODE REVIEW

**Comment by neverendingqs** [comment]
Preview looks good: https://5ce07ea766a1df000763e999--chesselo.netlify.com/

File: README.md [hunk 1]
```diff
@@ -10,5 +10,18 @@ Bob,Alice,black       | Bob was black, Alice white, and Alice won
...
+```sh
+# Run dev server
+npm run serve
+
+# Run tests
+npm test
+
+# Production build
+npm run build
+```
```
Comment (neverendingqs) [inline README.md]: Dev usage notes

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[build]
+  base    = "."
+  publish = "dist"
+  command = "npm run build"
```
Comment (neverendingqs) [inline netlify.toml]: Netlify config

**Comment by turtleman270** [comment]
> It's a bit heavier than a simple html/js/css, but less hacky. Thoughts?

I'm kind of just assuming that whatever you do is "right" lol

File: README.md (hunk 1 above)
Comment (turtleman270) [reply README.md]: When would I need to run `npm run build`?

Comment (neverendingqs) [reply README.md]: Probably never locally. Netlify runs it

**Pull request merged**